packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -293,8 +295,12 @@ export class EpochTracker implements IPersistedFileCache {
         fetchOptions.body = formParams.join("\r\n");
     }
 
-    private formatClientCorrelationId() {
-        return `driverId=${this.driverId}, RequestNumber=${this.networkCallNumber++}`;
+    private formatClientCorrelationId(fetchReason?: string) {
+        let clientCorrelationId = `driverId=${this.driverId}, RequestNumber=${this.networkCallNumber++}`;
+        if (fetchReason !== undefined) {
+            clientCorrelationId += `, fetchReason=${fetchReason}`;
```

Comment:
Suggestion - For this kind of thing I like to use string[].join, like:

```
const parts: string[] = [`driverId...`, `RequestNumber...`];
if (fetchReason !== undefined) { parts.push(`fetchReason...`) }
return parts.join(", ");
```

Reply:
PS - It's too late to change this thing's format to JSON right?

Reply:
Sounds good.